pages/index.vue

```diff
@@ -3,8 +3,20 @@
     <div class="element-wrapper">
       <div class="element-body">
         <h1>Welkom bij de Levarne Nulmeting</h1>
-        <LevButton @clicked="increment">Count me! {{ cnt }}</LevButton>
+        <LevButton @clicked="todoStore.callAPI">Haal opdracht op!</LevButton>
         <br>
+        <ul id="todo-table">
+          <li v-for="todo in todos" class="table-item">
+            {{ todo.assignee }} - {{ getNormalTimeFormat(todo.dueDateTime)}} - {{ todo.description }}
+          </li>
+        </ul>
+        <br>
+        <label>Kies een sorteer optie: </label>
+        <select ref="sortselect">
+          <option value="Name">Naam</option>
+          <option value="Date">Datum</option>
+          <option value="Description">Beschrijving</option>
+        </select>
```

Comment:
Deze kan je dmv v-model koppelen aan een variabele die in je <script setup> deel staat, en dan obv daarvan de todos computed property berekenen. Dan zul je zien dat als je een sorteer optie selecteerd, die computed property automatisch update 👀 

https://vuejs.org/guide/essentials/forms.html#text